src/index.html

```diff
@@ -10,13 +10,67 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="./style.css"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav-logo">
+        <a href="home">
+          <img
+            src="./images/logo.png"
+            alt="Moyo logo"
+            class="nav-bar-logo"
+          />
+        </a>
+      </div>
+      <nav class="nav-bar">
+        <div class="nav-menu">
+          <ul class="nav-items">
+            <li>
+              <a
+                href="apple"
+                class="is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li><a href="samsung">Samsung</a></li>
+            <li><a href="smartphones">Smartphones</a></li>
+            <li>
+              <a
+                href="Laptops"
```

Comment:
The `href` attribute value 'Laptops' should ideally be a valid URL or a more descriptive path, such as './laptops.html'.